app/code/Magento/Newsletter/Controller/Manage/Save.php

```diff
@@ -88,7 +94,7 @@ public function execute()
                     ->getParam('is_subscribed', false);
                 if ($isSubscribedParam !== $isSubscribedState) {
                     // No need to validate customer and customer address while saving subscription preferences
-                    $this->setIgnoreValidationFlag($customer);
+                    $customer->setData('ignore_validation_flag', true);
```

Comment:
What is the reason for this change?

Reply:
@melnikovi - It was trowing PHPMD coupling between objects error. so i checked and found that we don't need this method because it is private method and used only once in scope. Also there was no calculation in method.
If you wish i can revert it and remove PHPMD error by suppressing it